electrumx/server/block_processor.py

```diff
@@ -408,6 +408,17 @@ def advance_blocks(self, blocks):
         self.tip = self.coin.header_hash(headers[-1])
 
     def advance_txs(self, txs, is_unspendable):
+
+        def generate_staking_bytes(is_stake):
+            if is_stake:
+                return b'\x01\x00'
+            else:
+                return b'\x00\x00'
+
+        def unpac_staking_utxo_index(pk_script):
+            staking_idx, = unpack_le_uint32(pk_script[-2:-1]+b'\x00\x00\x00')
```

Comment:
This number is not a `uint32`, it's a `varint` or `compact_size` format. It is documented here: https://learnmeabitcoin.com/technical/varint

Reply:
fixed